src/Tuura/Boolean.hs

```diff
@@ -0,0 +1,86 @@
+module Tuura.Boolean (
+  module Tuura.Boolean.Parser,
+  CNF, DNF, Literal (..),
+  convertToCNF, genConcepts,
+  simplifyCNF, simplifyDNF, convertCNFtoDNF) where
+
+import Tuura.Boolean.Parser
+import Data.List
+import Data.List.Extra
+import Data.Foldable
+import Data.Maybe
+import Data.Ord
+
+type CNF a = [[Literal a]]
+
+type DNF a = [[Literal a]]
```

Comment:
I'm thinking, perhaps, it's better to wrap these into a newtype, so it's not possible to accidentally use one instead of the other. Something like `newtype CNF a = CNF { fromCNF :: [[Literal a]] }`.